apps/clientside-bot/test/functions.test.ts

```diff
@@ -0,0 +1,1070 @@
+import { createConnection, Connection } from "typeorm"
+import {
+	guildConfigChangedBanlists,
+	handleReport,
+	handleRevocation,
+	splitIntoGroups,
+} from "../src/utils/functions"
+import BotConfig from "../src/database/BotConfig"
+import Command from "../src/database/Command"
+import FAGCBan from "../src/database/FAGCBan"
+import InfoChannel from "../src/database/InfoChannel"
+import PrivateBan from "../src/database/PrivateBan"
+import Whitelist from "../src/database/Whitelist"
+import {
+	createFAGCCategory,
+	createFAGCCommunity,
+	createFAGCReport,
+	createFAGCRevocation,
+	createGuildConfig,
+	createPrivateban,
+	createTimes,
+	createWhitelist,
+	randomElementsFromArray,
+	reportIntoFAGCBan,
+} from "./utils"
+import { Category, Community, Report } from "fagc-api-types"
+import faker from "faker"
+
+describe("splitIntoGroups", () => {
```

Comment:
Should include a check for partically filled splits. For example 750 into 500 and 250.